README.rst

```diff
@@ -172,6 +172,32 @@ Here are is a list of open source solutions that I have tried in the past.
 
 * ``duplicity``: great backup tool, but does not provide a sync solution of any kind.
 
+Contributing
+------------
+
+Pull requests are welcome! Make sure you pass all the tests, Travis will tell you if you don't ;)
+
+Running locally
+```````````````
+
+You will need ``libmagic`` installed:
+
+::
+
+    brew install libmagic
```

Comment:
Maybe something like

`You will need ``libmagic`` installed. This is installed by default on most linux distributions but on MacOSX you need to install it with brew as follows:`

This part probably makes sense to put in the "Installation" section considering installing s4 over pip will still fail. I wonder if there is a way to solve this depedency 🤔

Reply:
Makes sense, I moved it to the Installation section